### PR TITLE
Onboard centos pkgs

### PR DESCRIPTION
**Pull request by dhodovsk**

### CODE REVIEW

File: centos_playground/pkg_survey/survey.py [hunk 1]
```diff
@@ -70,9 +66,7 @@ def do_mock_build(self):
         c = subprocess.run(['mock', '-r', 'centos-stream-x86_64', 'rebuild', self.srpm_path])
         if not c.returncode:
             return
-        err_log_file = f"mock_error_builds/{self.project_info['name']}.log"
-        shutil.copyfile('/var/lib/mock/centos-stream-x86_64/result/build.log', err_log_file)
-        self.pkg_res['error'] = f'mock build failed. More info in: {err_log_file}'
+        self.pkg_res['error'] = f'mock build failed'
```
Comment (lachmanfrantisek) [inline centos_playground/pkg_survey/survey.py:69]:
```suggestion
        self.pkg_res['error'] = 'mock build failed'
```

File: centos_playground/onboard/onboard.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            repo=self.pkg_name,
+            namespace='source-git',
+            description=f"Source git repo for {self.pkg_name}.\n"
+                        f"For more info see: http://packit.dev/docs/source-git/")
+
+        p.add_user('centosrcm', AccessLevel.maintain)
+        p.add_group('git-packit-team', AccessLevel.maintain)
+        gr = Repo(self.cps.src_dir)
+        gr.create_remote('packit', p.get_git_urls()['ssh'])
+        gr.git.push('packit', 'c8', tags=True)
```
Comment (lachmanfrantisek) [inline centos_playground/onboard/onboard.py:43]: You came from the same area as Tomáš, didn't you?...;)

Please, don't use the cryptic variable names or at least don't reuse them...

**Review by lachmanfrantisek — COMMENTED** [review]
Good job with that onboarding!

And it looks really cool to see OGR in action...;) The script is very clear!

File: centos_playground/onboard/onboard.py (hunk 2 above)
Comment (TomasTomecek) [reply centos_playground/onboard/onboard.py:43]: gr = git repo, pretty clear to me -_- :D

**Review by TomasTomecek — APPROVED** [review]
nice!

brief documentation would be very nice

File: centos_playground/onboard/onboard.py (hunk 2 above)
Comment (lachmanfrantisek) [reply centos_playground/onboard/onboard.py:43]: Yes, `self.cps.pkg_res.get('conditional_patch')` sounds very clear..;)

**Review by lachmanfrantisek — APPROVED** [review]
:tada:

**Pull request merged**